### DISCUSSION TITLE
'postinstall'' is not recognized as an internal or external command

### DISCUSSION THREAD

**Discussion by L0GI0** [graphcommerce-org/graphcommerce#1788] [Q&A]
Hi, got this error when trying to install dependencies. Tried both with yarn and npm. Can this app be run out of the box on Windows?

npm WARN deprecated uglify-es@3.3.9: support for ECMAScript is superseded by `uglify-js` as of v3.13.0

> @graphcommerce/magento-graphcms@5.1.1 postinstall
> is-monorepo '[pkgrun] postinstall' '[pkgrun] patch-package'

'postinstall'' is not recognized as an internal or external command,
operable program or batch file.

**Comment by sinafath** [comment]
I have the same issue

**Comment by kim-ep** [comment]
Hey! I've also faced this issue while trying to run `yarn` command on Windows.
I've replaced single quotes with escaped double quotes in `postinstall` command in `package.json` and it solved the problem:
```
"scripts": {
    ...
    "postinstall": "is-monorepo \"[pkgrun] postinstall\" \"[pkgrun] patch-package\"",
    ...
}
```
Probably it also makes sense to replace single quotes to escaped double quotes in all other commands in "scripts" block to prevent similar errors.

**Reply by magital-gmbh** [reply]
Thanks!
